libp2p/Service.cpp

```diff
@@ -150,6 +150,17 @@ void Service::heartBeat()
     });
 }
 
+/// reset whitelist: stop session which is not in whitelist
+void Service::resetWhitelist()
+{
+    std::cout << "reset white list start" << std::endl;
+    RecursiveGuard l(x_sessions);
+    std::this_thread::sleep_for(std::chrono::milliseconds(5000));
+
+    std::cout << "reset white list end" << std::endl;
```

Comment:
Do not use the standard output asap

Reply:
Reasons for standard output?

Reply:
Next PR is the true implements of this function.

Reply:
Done. All logic has been push here now.